packages/core/src/lib/TwaManifest.ts

```diff
@@ -56,12 +57,14 @@ const DEFAULT_APP_VERSION_NAME = DEFAULT_APP_VERSION_CODE.toString();
 const DEFAULT_SIGNING_KEY_PATH = './android.keystore';
 const DEFAULT_SIGNING_KEY_ALIAS = 'android';
 const DEFAULT_ENABLE_NOTIFICATIONS = false;
+const DEFAULT_ENABLE_LOCATIONS = false;
```

Comment:
Shouldn't this be `DEFAULT_ENABLE_LOCATION` or `DEFAULT_ENABLE_LOCATION_DELEGATION`?